From 857420a3271dc5bbddbe45a9e9878b9dc88fd143 Mon Sep 17 00:00:00 2001
From: Gustavo Zacarias <gustavo@zacarias.com.ar>
Date: Tue, 3 Mar 2015 14:56:17 -0300
Subject: [PATCH] sparc/clone.S: guard tcb-offsets.h include with RESET_PID

Status: upstream [uClibc]

Otherwise we have a broken scenario with non-threading builds.

Signed-off-by: Gustavo Zacarias <gustavo@zacarias.com.ar>
---
 libc/sysdeps/linux/sparc/clone.S | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/libc/sysdeps/linux/sparc/clone.S b/libc/sysdeps/linux/sparc/clone.S
index 8d9da5a..1d0e3e6 100644
--- a/libc/sysdeps/linux/sparc/clone.S
+++ b/libc/sysdeps/linux/sparc/clone.S
@@ -22,7 +22,9 @@
 
 #include <asm/errno.h>
 #include <asm/unistd.h>
+#ifdef RESET_PID
 #include <tcb-offsets.h>
+#endif
 #include <sysdep.h>
 
 #define CLONE_VM	0x00000100
-- 
2.0.5

